build/_build/Build.Steps.cs

```diff
@@ -0,0 +1,1002 @@
+        .Unlisted()
+        .Description("Compiles the managed code in the src directory")
+        .After(CreateRequiredDirectories)
+        .After(Restore)
+        .Executes(() =>
+        {
+            // Always AnyCPU
+            DotNetMSBuild(x => x
+                .SetTargetPath(MsBuildProject)
+                .SetConfiguration(BuildConfiguration)
+                .DisableRestore()
+                .SetTargets("BuildCsharpSrc")
+            );
+        });
+
+
+    Target CompileNativeTestsWindows => _ => _
+        .Unlisted()
+        .After(CompileNativeSrc)
+        .OnlyWhenStatic(() => IsWin)
+        .Executes(() =>
+        {
+            // If we're building for x64, build for x86 too
+            var platforms =
+                Equals(Platform, MSBuildTargetPlatform.x64)
+                    ? new[] { MSBuildTargetPlatform.x64, MSBuildTargetPlatform.x86 }
+                    : new[] { MSBuildTargetPlatform.x86 };
+
+            // Can't use dotnet msbuild, as needs to use the VS version of MSBuild
+            MSBuild(s => s
+                .SetTargetPath(MsBuildProject)
+                .SetConfiguration(BuildConfiguration)
+                .SetTargets("BuildCppTests")
+                .DisableRestore()
+                .SetMaxCpuCount(null)
+                .CombineWith(platforms, (m, platform) => m
+                    .SetTargetPlatform(platform)));
+        });
+
+    Target CompileNativeTestsLinux => _ => _
+        .Unlisted()
+        .After(CompileNativeSrc)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            Logger.Error("We don't currently run unit tests on Linux");
+        });
+
+    Target CompileNativeTests => _ => _
+        .Unlisted()
+        .Description("Compiles the native loader unit tests")
+        .DependsOn(CompileNativeTestsWindows)
+        .DependsOn(CompileNativeTestsLinux);
+
+
+    Target CopyIntegrationsJson => _ => _
+        .Unlisted()
+        .After(Clean)
+        .After(CreateRequiredDirectories)
+        .Executes(() =>
+        {
+            var source = RootDirectory / "integrations.json";
+            var dest = TracerHomeDirectory;
+
+            Logger.Info($"Copying '{source}' to '{dest}'");
+            CopyFileToDirectory(source, dest, FileExistsPolicy.OverwriteIfNewer);
+        });
+
+    Target PublishManagedProfiler => _ => _
+        .Unlisted()
+        .After(CompileManagedSrc)
+        .Executes(() =>
+        {
+            DotNetPublish(s => s
+                .SetProject(Solution.GetProject(Projects.ClrProfilerManaged))
+                .SetConfiguration(BuildConfiguration)
+                .EnableNoBuild()
+                .EnableNoRestore()
+                .CombineWith(TargetFrameworks, (p, framework) => p
+                    .SetFramework(framework)
+                    .SetOutput(TracerHomeDirectory / framework)));
+        });
+
+    Target PublishNativeProfilerWindows => _ => _
```

Comment:
We want to move away from using the word "profiler" to avoid confusion with the Continuous Profiler product, so we should rename some steps. It doesn't need to be in this PR, but at some point before switching over completely to Nuke and the new pipeline. We haven't agreed on new names yet, but they should distinguish between the native and managed components of the Tracer, instead of "profiler" (native) vs "tracer" (managed).

Reply:
Yep, no problem. Just need to decide what those names are 🙂

Reply:
Don't let this block this PR, though. I know the whole thing is still work in progress. Just something to keep in mind.